react_render/django/render.py

```diff
@@ -41,6 +41,7 @@ def render_props(self):
 
     def render_props_b64(self):
         if self.props:
+            self.json_encoder = json.JSONEncoder(separators=(',', ':')).encode
```

Comment:
Also this needs to be DjangoJSONEncoder which is passed. Which formats decimals, datetime etc

Reply:
Can you write me a test to demonstrate?